Tests/LeafKitTests/LeafTests.swift

```diff
@@ -195,69 +196,54 @@ final class LeafTests: XCTestCase {
         try XCTAssertEqual(render(template, ["namelist": [.string("Tanner")]]), expectedName)
         try XCTAssertEqual(render(template), expectedNoName)
     }
-//
-//    func testEscapeExtraneousBody() throws {
-//        let template = """
-//        extension #("User") \\{
-//
-//        }
-//        """
-//        let expected = """
-//        extension User {
-//
-//        }
-//        """
-//        try XCTAssertEqual(renderer.testRender(template, .null), expected)
-//    }
-//
-//
-//    func testEscapeTag() throws {
-//        let template = """
-//        #("foo") \\#("bar")
-//        """
-//        let expected = """
-//        foo #("bar")
-//        """
-//        try XCTAssertEqual(renderer.testRender(template, .null), expected)
-//    }
-//
-//    func testCount() throws {
-//        let template = """
-//        count: #count(array)
-//        """
-//        let expected = """
-//        count: 4
-//        """
-//        let context = TemplateData.dictionary(["array": .array([.null, .null, .null, .null])])
-//        try XCTAssertEqual(renderer.testRender(template, context), expected)
-//    }
-//
-//    func testNestedSet() throws {
-//        let template = """
-//        #if(a){#set("title"){A}}title: #get(title)
-//        """
-//        let expected = """
-//        title: A
-//        """
-//
-//        let context = TemplateData.dictionary(["a": .bool(true)])
-//        try XCTAssertEqual(renderer.testRender(template, context), expected)
-//    }
-//
-//    func testDateFormat() throws {
-//        let template = """
-//        Date: #date(foo, "yyyy-MM-dd")
-//        """
-//
-//        let expected = """
-//        Date: 1970-01-16
-//        """
-//
-//        let context = TemplateData.dictionary(["foo": .double(1_337_000)])
-//        try XCTAssertEqual(renderer.testRender(template, context), expected)
-//
-//    }
-//
+
+//    func testEscapeExtraneousBody()... removed as obviated by syntax change
+
+    func testEscapeTag() throws {
+        let template = """
+        #("foo") \\#("bar")
+        """
+        let expected = """
+        foo #("bar")
+        """
+        try XCTAssertEqual(render(template, [:]), expected)
+    }
+    
+    // TODO: Reimplement #count
+    func _testCount() throws {
+        let template = """
+        count: #count(array)
+        """
+        let expected = """
+        count: 4
+        """
+        try XCTAssertEqual(render(template, ["array": ["","","",""]]), expected)
+    }
+    
+    // TODO: Are set/get totally deprecated?
+    func _testNestedSet() throws {
+        let template = """
+        #if(a){#set("title"){A}}title: #get(title)
+        """
+        let expected = """
+        title: A
+        """
+        try XCTAssertEqual(render(template, ["a": true]), expected)
+    }
+
+    // TODO: Reimplement #date
```

Comment:
Yes definitely. Many people will be upset if we don't have this haha.

Reply:
Addressing/adding in future PR concentrating on Tags